apps/system/locales/system.en-US.properties

```diff
@@ -609,6 +609,24 @@ copy-link=Copy Link
 
 okCharsLeft = OK ({{n}})
 
+#Tracking Protection Notice
+
+# LOCALIZATION NOTE (tracking-notice-header): This string is used as
+# title of the dialog that explains this new feature to new users.
+tracking-notice-header=New! Tracking Protection
+# LOCALIZATION NOTE (tracking-notice-description): This string is used as
+# introduction to the new feature. Link to the feature description in mdn.
+tracking-notice-description=This new feature protects your browsing activity against potential trackers. <a class="learnMore"> Learn More </a>
+tracking-notice-learn-more-url=https://support.mozilla.org/en-US/products/firefox-os/privacy-and-security
```

Comment:
Let's not put the url in the localization file. If we just use the `https://support.mozilla.org/products/firefox-os/privacy-and-security`, mozilla's servers we redirect to the right place. Let's put the href directly in the markup.